transcom/mymove
# INT-B-20824 - CLOSEOUT COMPLETE status for PPM  on CONFIRM document review

## [B-20824](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A988544)

## Summary

**INTEGRATION**

This PR fixes issue where CLOSEOUT_COMPLETE  status was not being set when SC counselor CONFIRMS and submits during Review Documents process. Previously, AWAITING CUSTOMER status was set when any of the documents were REJECTED. This caused a workflow issue where it was allowing customer's to upload PPM documents N times and not show the Download Payment Packet download button.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Login as customer and create and submit a move with a PPM.
2. Login as SC  and edit move PPM. Fill in all required info  in orders section and submit move details. 
3. Login as customer and upload documentations via Upload PPM Documents link. Go through upload process and complete all required steps. Add Weight and/or expenses. Submit.
4. Login as SC and go to Review Documents for PPM. Reject all or some of the documents. Make sure at least one item is REJECTED. Confirm and submit. Upon submission, Move PPM will go to status of CLOSEOUT_COMPLETE status.
5. Login as customer. Customer shall see a View Customer Feedback and Download Payment Packet buttons. Make sure both buttons work.
![Screenshot 2024-08-19 at 1 16 15 PM](https://github.com/user-attachments/assets/7608d377-e292-4e2d-871d-6041b3bbee25)



### Frontend


### Backend


## Screenshots
